.env

```diff
@@ -10,5 +10,10 @@ VITE_VISUALIZE_DISABLED=
 # Waiting vite-env support dynamic base path
 VITE_BASE_PATH=
 VITE_REVIEW_IDENTITY_PROVIDER=
+
 # When VITE_TELEMETRY_DISABLED equals true we disable telemetry (default: enabled)
 VITE_TELEMETRY_DISABLED=
+# Override max delay to wait for before sending a batch (default: 1min)
```

Comment:
maybe precise the format of its value (in milliseconds), knowing that in the current comment you give a default value in minute)